### PR TITLE
Dread: Burenia Logic Revision Part 2

### PR DESCRIPTION
**Pull request by XenoWars**
Did my best with this. Lots of big rooms in this PR that bloated the line count. The following rooms are those affected:

- Burenia Hub to Dairon
- Main Hub Tower Top
- Main Hub Tower Middle
- Energy Recharge South
- Teleport to Ghavoran
- Early Gravity Speedboost Rooms 1+2

Added a few events for speedboost blocks to account for door rando. I also did my best to improve the area around the lower part of Hub to Dairon to make it less dangerous, but there's only so much that can be done.

### CODE REVIEW

File: randovania/games/dread/json_data/Burenia.txt [hunk 1]
```diff
@@ -1102,11 +1112,11 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
 > Upper Left Magnet Wall; Heals? False
   * Layers: default
   > Door to Underwater Horseshoe
-      Flash Shift or Spider Magnet or Slide Jump (Beginner) or Simple IBJ or Use Spin Boost
+      Any of the following:
+          Flash Shift or Spider Magnet or Slide Jump (Beginner) or Simple IBJ or Use Spin Boost
```
Comment (Isachu) [inline randovania/games/dread/json_data/Burenia.txt:1116]: Check if Single Wall Jump is possible, then add if necessary

File: randovania/games/dread/json_data/Burenia.txt [hunk 2]
```diff
@@ -1221,15 +1224,21 @@ Extra - asset_id: collision_camera_010
   * Layers: default
   * Morph Ball Tunnel to Teleport to Ghavoran/Tunnel to Main Hub Tower Middle (Upper)
   > Door to Teleport to Ghavoran (Plasma)
-      Grapple Beam or Gravity Suit or Spider Magnet or Water Bomb Jump (Beginner)
+      Any of the following:
+          Gravity Suit or Spider Magnet or Movement (Beginner) or Perform WBJ
```
Comment (Isachu) [inline randovania/games/dread/json_data/Burenia.txt:1228]: This requires morph as you're doing a coyote jump out of the morph tunnel
```suggestion
          Gravity Suit or Spider Magnet or Perform WBJ
          Morph Ball and Movement (Beginner)
```

File: randovania/games/dread/json_data/Burenia.txt [hunk 3]
```diff
@@ -1832,29 +1855,41 @@ Extra - asset_id: collision_camera_015
   * Layers: default
   > Door to Main Hub Tower Middle (Charge)
       Trivial
-  > Event - Blob from nearby (db_reg_aq_012)
-      Shoot Beam
+  > Door to Main Hub Tower Middle (Plasma)
+      All of the following:
+          Wave Beam and Disabled Door Lock Randomizer
+          Any of the following:
+              Gravity Suit or Spider Magnet or Movement (Beginner) or Perform WBJ
```
Comment (Isachu) [inline randovania/games/dread/json_data/Burenia.txt:1862]: As above, this requires morph as you're doing a coyote jump out of the morph tunnel
```suggestion
          Gravity Suit or Spider Magnet or Perform WBJ
          Morph Ball and Movement (Beginner)
```

File: randovania/games/dread/json_data/Burenia.txt [hunk 4]
```diff
@@ -2056,22 +2089,20 @@ Extra - asset_id: collision_camera_018
...
 > Water Bottom; Heals? False
   * Layers: default
   > Tile Group (POWERBEAM) 2
       Any of the following:
-          Space Jump
-          Spider Magnet and Use Spin Boost
-          Gravity Suit and Simple IBJ
-          Grapple Beam and Walljump (Beginner)
+          All of the following:
+              Gravity Suit
```
Comment (Isachu) [inline randovania/games/dread/json_data/Burenia.txt:2099]: Please double check if gravity suit is required to reach the magnet

File: randovania/games/dread/json_data/Burenia.txt [hunk 5]
```diff
@@ -2122,31 +2143,37 @@ Extra - asset_id: collision_camera_019
...
+          Spider Magnet and Perform WBJ
   > Slope Top
       Use Spin Boost
 
 > Slope Top; Heals? False
   * Layers: default
   > Dock to Early Gravity Speedboost Room 1 (Upper)
       Any of the following:
-          Space Jump
+          Flash Shift or Use Spin Boost
```
Comment (Isachu) [inline randovania/games/dread/json_data/Burenia.txt:2161]: Should Flash Shift require Beginner Movement/Wall Jump here (not sure which to put)? You have to do a quick respin to make it up after the flash

**Review by Isachu — CHANGES_REQUESTED** [review]

**Comment by XenoWars** [comment]
ignore the first three commits

**Review by Isachu — APPROVED** [review]
Looks good

**Pull request merged**